README.adoc

```diff
@@ -1,11 +1,12 @@
 = AsciiCSAND: Asciidoctor processor for CSA Normal Documents (CSAND)
 
-image:https://img.shields.io/gem/v/metanorma-csand.svg["Gem Version", link="https://rubygems.org/gems/metanorma-csand"]
-image:https://travis-ci.com/metanorma/metanorma-csand.svg["Build Status", link="https://travis-ci.com/metanorma/metanorma-csand"]
-image:https://ci.appveyor.com/api/projects/status/gs07fuoo1xhe1spe?svg=true["Appveyor Build Status", link="https://ci.appveyor.com/project/metanorma/metanorma-csand"]
+image:https://img.shields.io/gem/v/metanorma-csa.svg["Gem Version", link="https://rubygems.org/gems/metanorma-csa"]
```

Comment:
@ronaldtse @opoudjis probably we should not rename this till actual release, please let me know how do you think

Reply:
@CAMOBAP795 let's change it.